py/torch_tensorrt/fx/tracer/dispatch_tracer/aten_tracer.py

```diff
@@ -96,12 +99,17 @@ def dynamo_trace(
     aten_graph: bool,
     tracing_mode: str = "real",
     dynamo_config: Optional[DynamoConfig] = None,
-) -> Tuple[torch.fx.GraphModule, Set[Guard]]:
+) -> Tuple[torch.fx.GraphModule, Set]:
```

Comment:
The `Guard` specification here was removed since this file still needs to be imported even when Torch 1.13 is used (`torch_tensorrt.dump_build_info()` automatically imports this). We cannot import the `Guard` module with Torch 1.13 since the Dynamo import is disabled on 1.13.